e2e/optimism-stack.Dockerfile

```diff
@@ -20,38 +20,42 @@ ENV PATH="${PATH}:/root/.foundry/bin"
 
 RUN foundryup
 
-ARG OP_GETH_CACHE_BREAK=1
+ARG OP_GETH_CACHE_BREAK=4
```

Comment:
Was this intentionally changed?

Reply:
haha I thought no one would notice, sort of;  the value doesn't mean much (just an easy way to break the cache which you can override in the cli, I was being lazy)